zip-0207.rst

```diff
@@ -0,0 +1,277 @@
+recipient addresses per block instead of one, corresponding to the three funding streams contained within it.
+
+
+Motivation
+==========
+
+Since the launch of the Zcash network, the consensus rules have required that until the first block reward
+halving (at block 850,000), each block must send 20% of the block subsidy to a hard-coded transparent address.
+[#block-subsidy]_ This stream of 2.5-ZEC outputs (the value after the mining slow-start was completed) can be
+split into three logical funding streams: [#continued-funding]_
+
+- (73.7%) The Founders' Reward stream
+- (14.4%) The Zcash Foundation Endowment stream
+- (11.9%) The Zerocoin Electric Coin Company Strategic Reserve stream
+
+Modifying the consensus rules to allocate the 2.5 ZEC across three separate recipient addresses decouples
+these three funding streams organizationally, legally, and operationally. It further reinforces transparency
+as to the structure of the original Founders' Reward.
+
+
+Specification
+=============
+
+Funding streams
+---------------
+
+A funding stream is defined by a constant value per block (in zatoshis), a start height (inclusive), and an
+end height (exclusive). The start height MUST be after the end of the mining slow-start (to make specification
+of the value simpler).
+
+- To-do: Decide whether to use fractional value per block instead, which gets around the mining slow-start
+  problem (and halvings) but might make implementation more complex.
+
+An active funding stream at a given block height is defined as a funding stream for which the block height is
+less than its end height, but not less than its start height.
+
+Each funding stream has an associated set of recipient addresses. Each address is used for at most 17500
+blocks, creating a roughly-monthly sequence of funding periods. The address to be used for a given block
+height is defined as follows (using ``SlowStartShift`` as-defined in [#block-subsidy]_)::
+
+    AddressChangeInterval = 17500
+    FundingStream.AddressIndex(height) =
+        floor((height - FundingStream.StartHeight) / AddressChangeInterval)
+    Address(height) = FundingStream.Addresses[FundingStream.AddressIndex(height)]
+
+- To-do: adjust ``AddressChangeInterval``, taking #3672 and #3690 (both reducing block times) into account.
+
+Consensus rules
+---------------
+
+Prior to activation of the Blossom network upgrade, the existing consensus rule for payment of the original
+Founders' Reward is enforced. [#original-fr-consensus-rule]_
+
+Once the Blossom network upgrade activates:
+
+- The existing consensus rule [#original-fr-consensus-rule]_ is no longer active.
+- The coinbase transaction in each block MUST contain at least one output per active funding stream that pays
+  the stream's value to the stream's recipient address for the block's height.
+
+Stream definitions
+------------------
+
+The three logical funding streams described above each start at the Blossom activation height, and end at the
```

Comment:
Delete logical as these are now three separate streams.

Reply:
The three funding streams have actually always been distinct, but the "implementation" has been legal / organizational rather than consensus based. Therefore, I propose:

```suggestion
The three consensus-defined funding streams described above each start at the Blossom activation height, and end at the
```

I want to be able to say accurately things like "this consensus change transitioned these three funding streams from a legal/organizational construct into blockchain consensus rules." and having some way to distinguish the two cases helps.